From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-hugetlb-support-vm_uffd_ops-api-checkpatch-fixes
Date: Fri Jun 27 02:24:07 PM PDT 2025

WARNING: please, no space before tabs
#41: FILE: mm/hugetlb.c:5462:
+^I.uffd_features^I= ^I__VM_UFFD_FLAGS,$

WARNING: please, no space before tabs
#43: FILE: mm/hugetlb.c:5464:
+^I.uffd_ioctls^I= ^IBIT(_UFFDIO_COPY) |$

Cc: Peter Xu <peterx@redhat.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/hugetlb.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/mm/hugetlb.c~mm-hugetlb-support-vm_uffd_ops-api-checkpatch-fixes
+++ a/mm/hugetlb.c
@@ -5459,9 +5459,9 @@ static vm_fault_t hugetlb_vm_op_fault(st
 
 #ifdef CONFIG_USERFAULTFD
 static const vm_uffd_ops hugetlb_uffd_ops = {
-	.uffd_features	= 	__VM_UFFD_FLAGS,
+	.uffd_features	=	__VM_UFFD_FLAGS,
 	/* _UFFDIO_ZEROPAGE not supported */
-	.uffd_ioctls	= 	BIT(_UFFDIO_COPY) |
+	.uffd_ioctls	=	BIT(_UFFDIO_COPY) |
 				BIT(_UFFDIO_WRITEPROTECT) |
 				BIT(_UFFDIO_CONTINUE) |
 				BIT(_UFFDIO_POISON),
_
